packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -274,20 +274,24 @@ export function isTaggedTelemetryPropertyValue(x: any): x is ITaggedTelemetryPro
  */
 // eslint-disable-next-line @rushstack/no-new-null
 function filterValidTelemetryProps(x: any): TelemetryEventPropertyType | null {
+    if(isPrimitive(x)){
+        return x;
+    }
+    if (Array.isArray(x) && x.every((val) => isPrimitive(val))) {
+        return JSON.stringify(x);
+    }
+    return null;
+}
+
+function isPrimitive(x:any): boolean {
```

Comment:
nit: This also returns false for `null`.  At least mention this in a comment even if you don't update the name (that would be fine since it's not exported)